package.json

```diff
@@ -19,7 +19,10 @@
   },
   "standard": {
     "globals": [
-      "fetch"
+      "fetch",
+      "XMLHttpRequest",
+      "FormData",
+      "File"
```

Comment:
Do we need these globals? I would really like to get rid of them.

Reply:
You can put `window` in front of everything and since these are only globals defined in browsers, it's even more precise.

`window. XMLHttpRequest` instead of `XMLHttpRequest` and so on.
Then you can remove all globals.